package.json

```diff
@@ -47,6 +48,7 @@
     "websocket-stream": "^5.1.1"
   },
   "dependencies": {
+    "@types/node": "^8.10.0",
```

Comment:
I prefer `@types/node` to be a devDependency.

Reply:
You're right, that's my bad.